.github/workflows/spell-checker.yml

```
@@ -17,4 +17,4 @@ jobs:
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
         continue-on-error: true
-        
\ No newline at end of file
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)